android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -53,6 +53,7 @@
     private final String CODE_PUSH_TAG = "CodePush";
     private final String DOWNLOAD_PROGRESS_EVENT_NAME = "CodePushDownloadProgress";
     private final String RESOURCES_BUNDLE = "resources.arsc";
+    private final String REACT_DEV_BUNDLE_CACHE_FILE_NAME = "ReactNativeDevBundle.js";
```

Comment:
We should add a comment here that mentions that this file needs to be kept in sync with the equivalent constant in the React Native repo, and even point to the file/line that its located at.